modules/metering-install-operator.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-installing-metering.adoc
+
+[id="metering-install-operator_{context}"]
+= Installing the Metering Operator
+
+To install the Metering Operator first create a dedicated OpenShift project for
+Metering, and then install the Metering Operator from OperatorHub.
+
+.Procedure
+
+. In the {product-title} web console, click *Projects* -> *Create Project*.
+
+. Select a name for the project (we recommend openshift-metering or something similar), and click *Create*.
```

Comment:
I would double check that they can create this. Last I knew, you couldn't create a project starting with "openshift-", (but you could create a namespace with that name).

Reply:
@lamek, @bergerhoffer is correct. it should be `Administration > Namespaces > Create Namespace`

Reply:
We need to switch to create namespace and only using `openshift-metering` as the namespace across the board for our GA. No other namespace will be supported. Additionally, I realize we need to tell them to label the namespace too so that monitoring is enabled for our components.

```
oc label  namespace openshift-metering 'openshift.io/cluster-monitoring=true'
```